tests/testapp/urls.py

```diff
@@ -51,4 +51,5 @@
     path('template/<int:pk>/', views.template_detail, name='test_template_detail',),
     path('library/', views.library_list, name='test_library_list',),
     path('library/<int:pk>/', views.library_detail, name='test_library_detail',),
+    path('booknestedshelf/', views.book_nested_shelf, name='test_book_nested_shelf',),
```

Comment:
Okay, for this one, why did we need another endpoint?

Reply:
This error is raised only with the serializer classes having nested serializer field. Also see:- https://github.com/openwisp/openwisp-users/pull/260#discussion_r652607748

Reply:
Okay, can you please go to this endpoint and try to create a book nested shelf?
I am not able to do that...
```
{
    "shelf": [
        "This field cannot be null."
    ]
}
```

Reply:
https://github.com/openwisp/openwisp-users/pull/260/files#r659296852